target/linux/mediatek/filogic/base-files/etc/hotplug.d/ieee80211/11_fix_wifi_mac

```diff
@@ -55,15 +55,8 @@ case "$board" in
 		[ "$PHYNBR" = "1" ] && macaddr_setbit_la $(macaddr_add $addr 2) > /sys${DEVPATH}/macaddress
 		;;
 	cmcc,rax3000m)
-		case "$(cmdline_get_var root)" in
-		/dev/mmc*)
-			addr=$(mmc_get_mac_binary factory 0xa)
-			;;
-		*)
-			addr=$(mtd_get_mac_binary factory 0xa)
-			;;
-		esac
-		[ "$PHYNBR" = "1" ] && echo "$addr" > /sys${DEVPATH}/macaddress
+		addr=$(cat /sys/class/net/eth0/address)
+		[ "$PHYNBR" = "1" ] && macaddr_add $addr -1 > /sys${DEVPATH}/macaddress
```

Comment:
How annoying, they really use `-1` offset for GMAC1... Ok...

Reply:
Nah, gmac1 is gmac0 `-3`.
MAC table on this board:
| dev | mac | offset |
| :---: | :----: | :----: |
| GMAC0 (LAN) | B8:60:61:74:XX:64 | / |
| GMAC1 (WAN) | B8:60:61:74:XX:61 | -3 |
| WLAN2G | B8:60:61:74:XX:62 | -2 |
| WLAN5G | B8:60:61:74:XX:63 | -1 |